megamek/src/megamek/client/ui/swing/tooltip/UnitToolTip.java

```diff
@@ -495,37 +616,77 @@ private static StringBuilder weaponList(Entity entity) {
                 nameStr = nameStr.replace(CLANWP, "");
                 nameStr += currentEquip.range;
 
-                result.append(guiScaledFontHTML(uiTTWeaponColor()));
+                result.append(guiScaledFontHTML(uiTTWeaponColor()) + "\n");
                 String techBase = "";
                 if (entity.isMixedTech()) {
-                    techBase = currentEquip.isClan ? Messages.getString("BoardView1.Tooltip.Clan") :
-                            Messages.getString("BoardView1.Tooltip.IS");
+                    String msg_clan = Messages.getString("BoardView1.Tooltip.Clan");
+                    String msg_is = Messages.getString("BoardView1.Tooltip.IS");
+                    techBase = currentEquip.isClan ? msg_clan : msg_is;
                     techBase += " ";
                 }
-                String destStr = isDestroyed ? "<S>" : "";
+
+                String msg_isDestoryed= Messages.getString("BoardView1.Tooltip.IsDestroyed");
+                String destStr = isDestroyed ? msg_isDestoryed : "";
 
                 if (totalWeaponCount > 5 && hasMultiples) {
                     // more than 5 weapons: group and list with a multiplier "4 x Small Laser"
-                    result.append("<TR><TD>");
+                    result.append("<TR>\n<TD>\n");
                     if (currentEquip.count > 1) {
-                        result.append(currentEquip.count + " x ");
+                        String msg_x = Messages.getString("BoardView1.Tooltip.X");
+                        result.append(currentEquip.count + " " + msg_x + " ");
                     }
-                    result.append("</TD><TD>");
+                    result.append("\n</TD>\n<TD>\n");
                     result.append(addToTT("Weapon", false, currentEquip.count, techBase, nameStr, destStr));
                     result.append(weaponModifier(isDestroyed, currentEquip));
-                    result.append("</TD></TR>");
+                    result.append("\n</TD>\n</TR>\n");
                 } else {
                     // few weapons: list each weapon separately
                     for (int i = 0; i < currentEquip.count; i++) {
-                        result.append("<TR><TD></TD><TD>");
+                        result.append("<TR>\n<TD>\n</TD>\n<TD>\n");
                         result.append(addToTT("Weapon", false, currentEquip.count, techBase, nameStr, destStr));
                         result.append(weaponModifier(isDestroyed, currentEquip));
-                        result.append("</TD></TR>");
+                        result.append("\n</TD>\n</TR>\n");
                     }
                 }
+                result.append("</FONT>\n");
             }
         }
-        result.append("</TABLE>");
+        result.append("</TABLE>\n");
+        return result;
+    }
+
+    private static StringBuilder bombList(Entity entity) {
+        StringBuilder result = new StringBuilder();
+
+        if (entity.isBomber()) {

```

Comment:
Looks to me like l.661 could already be the instanceof check and the in l.665 could be omitted.